app/tests/admin/test_onboarding.py

```diff
@@ -70,27 +72,12 @@ async def test_onboard_issuer_public_did_exists(
         )
     )
     onboard_result = await onboarding.onboard_issuer(
-        name="issuer_name",
```

Comment:
Why did you remove the name? This is used in the trust registry I think

Reply:
also left over from debugging with different versions

Reply:
will put it back